engines/config-query-sparql/config/dereference/actors.json

```diff
@@ -12,7 +12,7 @@
       "@id": "urn:comunica:default:dereference/actors#http",
       "@type": "ActorDereferenceHttp",
       "beforeActors": { "@id": "urn:comunica:default:dereference/actors#fallback" },
-      "mediatorHttp": { "@id": "urn:comunica:default:http/mediators#main" }
+      "mediatorHttp": { "@id": "urn:comunica:default:http-intercept/mediators#main" }
```

Comment:
I'm surprised this works. Wondering if it shouldn't?

Because the dereference-http actor is expecting a mediator over the HTTP bus, but it is actually receiving a mediator over the HTTP-intercept bus, which is a different thing.
So perhaps we need to change this param to `mediatorHttpIntercept`, and make it explicit typing-wise as well?

Related to this, we have several other actors that are using the HTTP mediator as well. Should we change this to this interceptor as well?

If we make these changes, this will be a breaking change, which we may not want to do (again) at this stage.

Reply:
I think the wrapper-based approach discussed earlier might lead to less breakage.

Reply:
Not quite sure I understand what you mean by wrapper approach - and how it differs to what I've done here. Regardless I've cleaned all this up in #1077 by using two `http` bus' in the build (which I only just worked out how to do) rather than introducing an intercept `-bus`